sdmetrics/column_pairs/statistical/inter_row_msas.py

```diff
@@ -29,7 +29,61 @@ class InterRowMSAS:
     max_value = 1.0
 
     @staticmethod
-    def compute(real_data, synthetic_data, n_rows_diff=1, apply_log=False):
+    def _validate_inputs(real_data, synthetic_data, n_rows_diff, apply_log):
+        for data in [real_data, synthetic_data]:
+            if (
+                not isinstance(data, tuple)
+                or len(data) != 2
+                or (not (isinstance(data[0], pd.Series) and isinstance(data[1], pd.Series)))
+            ):
+                raise ValueError('The data must be a tuple of two pandas series.')
```

Comment:
Should we added tests for these 3 exceptions?

Reply:
The [tests](https://github.com/sdv-dev/SDMetrics/blob/main/tests/unit/column_pairs/statistical/test_inter_row_msas.py#L92-L156) have already been implemented. This change simply reorganizes the function for readability.